tests/network-ovn

```diff
@@ -2124,9 +2122,7 @@ else
     ovn_leases_tests
 fi
 
-lxc image delete "${instanceImage}"
-lxc config unset storage.images_volume
-lxc storage volume delete default images
+lxc profile device remove default root
```

Comment:
I don't understand why removing the `root` device from the default profile wasn't needed before? @tomponline any idea?

Reply:
Before your recent set of changes we werent adding the root disk device to the profile but instead creating the instances directly using the `-s` flag to add a root disk to each instance directly so there was less clean up needed.

Reply:
The root disk device as long been in the default project (since commit 88252e0d). What I did change is the `features.profiles` on some projects but it's still unclear to me what I introduced that needed that additional cleanup?

Looking at `git diff --follow  -p 88252e0d.. -- tests/network-ovn`, I'm not seeing anything that could explain this. Anyway, not something important but I was (still am) curious. Thanks!